composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "yarcode/yii2-free-kassa",
+  "name": "nicdnepr/yii2-free-kassa",
```

Comment:
it's added automatically
just this https://github.com/yarcode/yii2-free-kassa/pull/1/commits/a66f11793edf7a43513524ad74bfc8df926da74e